### ISSUE TITLE
Add basic editing for text files

### ISSUE THREAD

**Issue by hosy** [owncloud/ios-app#317]
Text files should be editable 

### Steps to reproduce
1. Open text file

### Expected behaviour
Text content should be editable inside the app

### Actual behaviour
File content is preview only

### Client
**iOS version:**
12.1.4

**ownCloud app version:**
1.0.0 build 111

**Comment by michaelstingl** [comment]
No, with iOS Files integration, we don't need text editing inside the app. There are much better apps for this that integrate nicely. (see https://github.com/owncloud/ios-app/issues/68#issuecomment-395715378)

**Comment by michaelstingl** [comment]
Old issues from encoding hell:

- https://github.com/owncloud/ios/issues/949
- https://github.com/owncloud/ios/issues/442
- https://github.com/owncloud/ios/issues/368
- https://github.com/owncloud/ios/issues/113

**Comment by michaelstingl** [comment]
@hosy is working on a POC:
- based on a popular library https://github.com/simonbs/Runestone

**Comment by stackByte** [comment]
Sounds like a pretty basic feature that MUST be available.  This surely will add to the productivity capability.  If there's a vote, I would put my hand up for it.
